MoviesExample/app/src/main/res/layout/movies_list_card.xml

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/tools">
```

Comment:
Shouldn't this be `http://schemas.android.com/apk/res-auto`?
Currently this causes the build to break:
```
Error: Suspicious namespace and prefix combination [NamespaceTypo]
  <layout xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/tools"
```

Reply:
Fixed in ce658fd
(sorry AS also re-formatted the file)